index.html

```diff
@@ -4711,9 +4711,8 @@ <h3>Proofs (Signatures)</h3>
 The cryptographic mechanism used to prove that the information in a
 <a>verifiable credential</a> or <a>verifiable presentation</a> was not
 tampered with is called a <em>proof</em>. There are many types of cryptographic
-proofs including, but not limited to, digital signatures, zero-knowledge proofs,
-Proofs of Work, and Proofs of Stake. In general, when verifying proofs,
-implementations are expected to ensure:
+proofs including, but not limited to, digital signatures, zero-knowledge proofs. 
```

Comment:
```suggestion
proofs including, but not limited to, digital signatures & zero-knowledge proofs. 
```

Reply:
Noting that we have yet to document any credential proof formats that use zero knowledge proofs.